public/components/agents/vuls/inventory/lib/api-requests.ts

```diff
@@ -22,6 +27,15 @@ export async function getFilterValues(field, value, agentId, filters={}, format=
   if (value) {
     filter['search'] = value;
   }
-  const result = await WzRequest.apiReq('GET', `/vulnerability/${agentId}`, { params: filter });
-  return (((result || {}).data || {}).data || {}).affected_items.map((item) => {return format(item[field])});
-}
\ No newline at end of file
+  const result = await WzRequest.apiReq('GET', `/vulnerability/${agentId}`, { params: filter });
+  return (((result || {}).data || {}).data || {}).affected_items.map((item) => { return format(item[field]) });
```

Comment:
**nitpick**: We could use the `optional chaining` operator (`?.`) because of the typescript support.

```ts
return result?.data?.data?.affected_items?.map((item) => format(item[field]));
```